app/javascript/components/rooms/RoomView.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios"
+import {Row, Col, Tabs, Tab} from "react-bootstrap";
+import ButtonLink from "../stylings/buttons/ButtonLink";
+import FeatureTabs from "./FeatureTabs";
+import {Link, useParams} from "react-router-dom";
+import {useQuery} from "react-query";
+import { Spinner } from "../stylings/Spinner"
+import {House} from "react-bootstrap-icons";
+
+export default function RoomView() {
+  const { friendly_id } = useParams()
+
+  const { isLoading, error, data, isFetching } = useQuery("getRoom", () =>
+    axios.get(`/api/v1/rooms/${friendly_id}.json`, {
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+      }
+    }).then(resp => resp.data)
```

Comment:
Not sure what approach we want to use for these since I know they will be moved to another file soon, but you can do something like this (using useState) which would allow you to just access room attributes easier:
```suggestion
    }).then(function (resp) {
      setRoom(resp.data.data)
    })
```

Then you'd be able to do room.name instead of data.data.name.

But then 'data' returned from useQuery won't really be used anymore, so I am not sure what is best.